app/src/main/java/io/apicurio/registry/mt/TenantManagerClientProducer.java

```diff
@@ -75,12 +75,13 @@
                             "but the no auth properties aren't properly configured");
                 }
 
+                ApicurioHttpClient httpClient = new JdkHttpClientProvider().create(properties.getTenantManagerAuthUrl().get(), Collections.emptyMap(), null, new AuthErrorHandler());
```

Comment:
the same here, I know it's out of the scope. but an improvement would be to share the http client between oidcauth and tenant manager client. I know this may be complex so no worries we can do it in the future